bundle/manifests/multicluster-observability-operator.clusterserviceversion.yaml

```diff
@@ -58,17 +58,17 @@ spec:
   customresourcedefinitions:
     owned:
     - description: MultiClusterObservability defines the configuration for the Observability installation on Hub and Managed Clusters all through this one custom resource.
-      displayName: Multi Cluster Observability
```

Comment:
How does it generate?

Reply:
Looks like it automatically divide the word by camel case, not sure about that.